pkg/networkengine/routedriver/vxlan/vxlan.go

```diff
@@ -244,6 +245,17 @@ func (vx *vxlan) ensureRavenMarkChain() error {
 	if err := vx.iptables.AppendIfNotExists(iptablesutil.MangleTable, iptablesutil.OutputChain, "-j", iptablesutil.RavenMarkChain); err != nil {
 		return fmt.Errorf("error adding chain %s rule: %s", iptablesutil.OutputChain, err)
 	}
+	// for raven skip nat
+	if err := vx.iptables.NewChainIfNotExist(iptablesutil.NatTable, iptablesutil.RavenPostRoutingChain); err != nil {
+		return fmt.Errorf("error create %s chain: %s", iptablesutil.RavenPostRoutingChain, err)
+	}
+	if err := vx.iptables.InsertIfNotExists(iptablesutil.NatTable, iptablesutil.PostRoutingChain, 1, "-o", "raven0", "-j", iptablesutil.RavenPostRoutingChain); err != nil {
+		return fmt.Errorf("error adding chain %s rule: %s", iptablesutil.PostRoutingChain, err)
+	}
+	if err := vx.iptables.AppendIfNotExists(iptablesutil.NatTable, iptablesutil.RavenPostRoutingChain, "-j", "RETURN"); err != nil {
```

Comment:
Use Accept target to skip SNAT on follow POSTROUTING chain.

Reply:
done